exercises/tmclass_exercises/text_manipulation.py

```diff
@@ -83,6 +90,5 @@ def tokenize_japanese(text):
     # - Use the `janome.tokenizer.Tokenizer` class tokenize the text
     # - Read the online documentation of the janome package to only return
     #   the surface form for each token.
-
-    # TODO: write me!
-    return []
```

Comment:
Please delete this "return" or comment it.

Reply:
thank for your remark